<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>420</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>YouTube Download</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>47</y>
      <width>420</width>
      <height>5</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="thumbnailL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>400</width>
      <height>225</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid rgb(179, 176, 174);
border-radius: 2%;
</string>
    </property>
    <property name="text">
     <string>Thumbnail Preview</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="statusL">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>380</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress Status:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressPB">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>355</y>
      <width>360</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTreeWidget" name="streamsTW">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Streams</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter directory path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="directoryL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>155</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 1px inset rgb(179, 176, 174);
border-radius: 2%;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>50</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter YouTube URL:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadurlPB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelPB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadPB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Download stream(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageL">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>360</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
